.eslintrc.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
     'plugin:promise/recommended',
     'plugin:import/errors',
     'plugin:import/warnings',
-    'plugin:import/typescript'
+    'plugin:import/typescript',
+    'plugin:vue/recommended'
```

Comment:
@ferferga Vue stuff should go in the override below. I'm also fairly sure this was already implied by some of the configs we extend, but I'm not fully sure.